src/util/std_types.h

```diff
@@ -79,7 +79,7 @@ class symbol_typet:public typet
 
 /// Check whether a reference to a typet is a \ref symbol_typet.
 /// \param type Source type.
-/// \return True if \param type is a \ref symbol_typet.
+/// \return True if type is a \ref symbol_typet.
```

Comment:
Ok for correct syntax, but is there some way to make Doxygen understand that this "type" refers to the named parameter "type"? I believe this is what the author intended.

Reply:
I couldn't find a way to explicitly link it to the parameter.  I tried adding emphasis, or marking it as code, but I think it looks best as plain text.

Reply:
How about using `\p`?

Reply:
I hadn't seen that, thanks.  It certainly looks from the Doxygen documentation like that is the way to go (https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdp).  Their example looks good:

![screen shot 2018-10-15 at 14 25 02](https://user-images.githubusercontent.com/32836213/46953604-364cd480-d086-11e8-8d10-6dbb27e63eee.png)

However, it doesn't look so good in the documentation generated for cbmc:

![screen shot 2018-10-15 at 14 20 51](https://user-images.githubusercontent.com/32836213/46953468-dce4a580-d085-11e8-849d-511c82e73b2e.png)

I guess it's probably best to use the \p in the cbmc documentation now, and then possibly (in future) consider changing the cbmc documentation formatting so that it is more like the standard Doxygen formatting.